docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,131 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained_ and unofficial_ forks of project documentation.
+
+
+Rationale
+---------
+
+Documentation projects may be kept online indefinitely, even though a newer version of the same project exists elsewhere.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in search results,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow project maintainers to request the *delisting* of forks that are old and outdated.
```

Comment:
I am quite okay about removing "maintainers". I think anyone can start this process and a request to remove something can always be taken into consideration. For instance, a user who has an extremely bad experience from reading the wrong documentation.

Also, removing this saves us the trouble of verifying if the person in question is a maintainer of said project.

Reply:
I tend to agree here. I don't think it _must be one of the maintainers_ the ones that can request it.

Reply:
We have a step now in our process (suggested by @agjohnson ) where we alert maintainers of the original project of any delistings made. While we could scratch that step since it doesn't affect any of the preceding actions, it seems like a nice gesture that is helpful to the overall community.

Reply:
If it changes anything here, I was more concerned with contacting the maintainers of the project we're blocking from search.

It's probably going to be something like 95% of requests to delist a fork will indeed come from the actual maintainers of the repostory/package/etc, so we'll probably already have a line of communication open with them. If it is a community report though, I probably wouldn't feel much need to reach out to the maintainers of the official repo/project/package/etc.

Reply:
So I suppose I also agree with the above, swapping "project maintainers" for something more generic.

Reply:
> I was more concerned with contacting the maintainers of the project we're blocking from search.

Isn't that what the reachability step is about?

(but yes, I did add a suggestion of an ultimate step to inform the maintainers of the "original" project)

Reply:
I'll try to introduce a definition of "source project", so in these cases we can refer to "source project maintainers".

Reply:
> Isn't that what the reachability step is about?

I was describing skipping the reachability step with my comment here:

https://github.com/readthedocs/readthedocs.org/pull/9731#discussion_r1023171957

Reply:
Now I understand https://github.com/readthedocs/readthedocs.org/pull/9731#discussion_r1023171957 much better. I think that defining the scenario for skipping reachability can be achieved. Probably it can be fixed by adding a definition of _when_ reachability is necessary or unnecessary (whichever definition is simplest).

For instance, in the case of an unofficial documentation project that has had significant efforts placed but is still unmaintained and unofficial? Like, if we can identify that a significant fraction of the documentation content is original, we should try to reach the documentation project's maintainer(s).

Reply:
> For instance, in the case of an unofficial documentation project that has had significant efforts placed but is still unmaintained and unofficial? Like, if we can identify that a significant fraction of the documentation content is original, we should try to reach the documentation project's maintainer(s).

Yea, if there is activity in the last 6 months presumably it's not unmaintained, and a bigger question.

Reply:
I like that. It means that if an unofficial documentation set doesn't receive updates for 6 months and someone reports it for whatever reason, it means that it _can_ be unlisted. But its maintainers will also be notified. So at the end of the day, they may have a response that can make bring their efforts back to "maintained".

Reply:
> So I suppose I also agree with the above, swapping "project maintainers" for something more generic.

I have now removed mentioning of "who" can request a delisting.

I didn't find more ambiguous mentioning of "maintainer". I changed all occurrences of "owner" with "owners", I believe we can define what an "owner" is -- it is most likely the entire maintainer team.